report_scripts/patient_status_event.sql

```diff
@@ -0,0 +1,86 @@
+DROP PROCEDURE IF EXISTS patient_status_event;
+DELIMITER $$
+CREATE PROCEDURE patient_status_event(IN org_unit VARCHAR(11))
```

Comment:
I think we do not use org_unit as procedure parameter.

Reply:
It came from wrong auto-merge after renaming this file and changing it on a different branch. I'll remove it.